helmfile.d/values/_helpers.tpl

```diff
@@ -90,3 +90,11 @@ after the separator.
 {{- define "gen.container_uri" -}}
 {{ with include "gen.reg-rep-img" . }}{{ . }}{{ end }}{{ if or .tag .digest }}:{{ .tag }}{{ if .digest }}@{{ .digest }}{{ end }}{{ end }}
 {{- end }}
+
+{{- define "componentObjectStorage" -}}
+{{- $component := index . 0 -}}
+{{- $key := index . 1 -}}
+{{- $ctx := index . 2 -}}
+{{- $default := $ctx.Values | dig "objectStorage" "s3" $key "" -}}
+{{- $ctx.Values | dig $component "objectStorage" "s3" $key $default | quote }}
+{{- end }}
```

Comment:
Just to make sure I'm reading this correctly:
By default it would take the regular `objectStorage.s3.accessKey` and `objectStorage.s3.secretKey`, but with this it is also possible to use `harbor.objectStorage.s3.accessKey` and `harbor.objectStorage.s3.secretKey`?